database/sql/V093__create_cow_amm_table.sql

```diff
@@ -0,0 +1,7 @@
+-- Create table to store information about indexed CoW AMMs
+CREATE TABLE cow_amms (
+    address BYTEA NOT NULL PRIMARY KEY,
+    factory_address BYTEA NOT NULL,
+    tradeable_tokens BYTEA[] NOT NULL,
+    block_number BIGINT NOT NULL
```

Comment:
Could we please also store the tx_hash here? A tx_hash is often quite useful and we have some tables that are annoying to use due to the lack of `tx_hash`. Probably less relevant for cow amms but would be good to still learn from past mistakes.